config/node.ci.options

```diff
@@ -0,0 +1,5 @@
+## Node command line options for CI
```

Comment:
nice! thoughts on node.dev.options?  i'm almost permanently increasing memory limits

Reply:
it's there already, my bad